src/test/java/uk/gov/ons/census/casesvc/service/CaseServiceTest.java

```diff
@@ -228,13 +227,12 @@ public void testCreateCCSCaseWithRefusalReceived() {
             });
 
     // When
-    Case actualCase = underTest.createCCSCase(caseId, sampleUnit, HARD_REFUSAL, false);
+    Case actualCase = underTest.createCCSCase(caseId, sampleUnit);
```

Comment:
If we're no longer dealing with refusals in CCS case events then I think this test can go?